elements/lisk-transactions/src/12_multisignature_transaction.ts

```diff
@@ -361,4 +372,69 @@ export class MultisignatureTransaction extends BaseTransaction {
 
 		return createResponse(this.id, []);
 	}
+
+	public signAll(
+		networkIdentifier: string,
+		senderPassphrase: string,
+		keys?: {
+			readonly passphrases: ReadonlyArray<string>;
+			readonly mandatoryKeys: ReadonlyArray<string>;
+			readonly optionalKeys: ReadonlyArray<string>;
+			readonly numberOfSignatures: number;
+		},
+	): void {
+		// Set network identifier if it was previously not set in the transaction
+		if (!this._networkIdentifier) {
+			this._networkIdentifier = networkIdentifier;
+		}
+		// Sort the keys in the transaction
+		sortKeysAscending(this.asset.mandatoryKeys);
+		sortKeysAscending(this.asset.optionalKeys);
+
+		// Sign with sender
+		const { publicKey } = getAddressAndPublicKeyFromPassphrase(
+			senderPassphrase,
+		);
+
+		if (this.senderPublicKey !== '' && this.senderPublicKey !== publicKey) {
+			throw new Error(
+				'Transaction senderPublicKey does not match public key from passphrase',
+			);
+		}
+
+		this.senderPublicKey = publicKey;
+
+		const networkIdentifierBytes = hexToBuffer(this._networkIdentifier);
+		const transactionWithNetworkIdentifierBytes = Buffer.concat([
+			networkIdentifierBytes,
+			this.getBytes(),
```

Comment:
didn't we change this to `getBasicBytes`?

Reply:
Fixed!